cmd/podman/containers/pause.go

```diff
@@ -65,12 +63,6 @@ func pause(cmd *cobra.Command, args []string) error {
 	var (
 		errs utils.OutputErrors
 	)
-	if rootless.IsRootless() && !registry.IsRemote() {
```

Comment:
After these changes - we still display an error on pausing a container in Cgroups v1 rootless, right?

Reply:
There is already a check in `(c *Container) pause()` so it should still error out correctly.

Reply:
Yes, I checked that an error message `Error: can not pause containers on rootless containers with cgroup V1: this container does not have a cgroup` is displayed in cgroups v1 rootless environment.